generate-stackbrew-pr.sh

```diff
@@ -76,7 +76,7 @@ function pr_payload() {
   local body
   local maintainers
 
-  escaped_message="$(echo "${COMMIT_MESSAGE}" | sed -E -e "s/\"/\\\\\"/g")"
+  escaped_message="$(echo "${COMMIT_MESSAGE}" | sed -E -e "s/\"/\\\\\"/g" | head -n1)"
```

Comment:
Let's keep the styling like the one in generate-stackbrew-pr.sh?
```diff
-    escaped_message="$(echo "${COMMIT_MESSAGE}" | sed -E -e "s/\"/\\\\\"/g" | head -n1)"
+    escaped_message="$(echo "${COMMIT_MESSAGE}" | sed -E -e "s/\"/\\\\\"/g" | head -n 1)"
```

Reply:
and maybe can put `head` before `sed`?

Reply:
Yeah, makes sense